src/roiextractors/extraction_tools.py

```diff
@@ -593,15 +593,15 @@ def animate_func(i, imaging, im, ax):
     return anim
 
 
-def check_keys(dict):
+def check_keys(dict_) -> dict:
```

Comment:
```suggestion
def check_keys(dict_: dict) -> dict:
```

This makes the conflation against the builtin a bit clearer

Reply:
(already in docstring)